forge-gui/res/cardsfolder/s/silvanuss_invoker.txt

```diff
@@ -0,0 +1,7 @@
+Name:Silvanus's Invoker
+ManaCost:2 G
+Types:Creature Dragon Druid
+PT:3/2
+A:AB$ Untap | Cost$ 8 | ValidTgts$ Land | TgtPrompt$ Select target land | SubAbility$ DBAnimate | PrecostDesc$ Conjure Elemental - | SpellDescription$ Untap target land you control. It becomes an 8/8 Elemental creature with trample and haste until end of turn. It's still a land.
```

Comment:
YouCtrl missing

Reply:
Ok. Does "YouCtrl"go on the animate line?

Reply:
`ValidTgts$ Land.YouCtrl`

Reply:
Got it! I'll get her changed. Thanks

Reply:
You'll want to clarify `TgtPrompt` along with that (if you haven't already)